src/controller.cpp

```diff
@@ -124,6 +129,32 @@ int Controller::run(const CliArgsParser& args)
 		return args.return_code().value();
 	}
 
+	if (!can_open(configpaths.url_file())) {
+		if (!args.silent()) {
+			std::cout << strprintf::fmt(_("Starting %s %s..."),
+					PROGRAM_NAME,
+					utils::program_version())
+				<< std::endl;
+
+			std::cout << '\n';
+
+			const auto locations = configpaths.expected_urls_paths();
+			std::cout << strprintf::fmt(
+					_("'urls' file not found. Please create it at one "
+						"of the following locations:\n"
+						"- %s\n"
+						"- %s\n"
+						"\n"
+						"Newsboat will store the rest of the files "
+						"accordingly."),
+					locations.first,
+					locations.second)
+				<< std::endl;
+		}
+
+		return EXIT_FAILURE;
```

Comment:
This breaks the automatic migration from Newsbeuter.

The migration is only executed if the Newsboat urls file does not exist.
But, if the urls file does not exist, this line will make sure that the following line below is not reached:
`const auto migrated = configpaths.try_migrate_from_newsbeuter();`

Reply:
Oh, that's embarrassing! The fix was easy though. This time, I did check that migrations do work as intended.

Thanks for spotting this!